prettier.config.js

```diff
@@ -0,0 +1,52 @@
+export default {
+	arrowParens: 'always',
+	bracketSpacing: true,
+	endOfLine: 'lf',
+	htmlWhitespaceSensitivity: 'css',
+	insertPragma: false,
+	singleAttributePerLine: false,
+	bracketSameLine: false,
+	printWidth: 120,
+	proseWrap: 'preserve',
+	quoteProps: 'as-needed',
+	requirePragma: false,
+	semi: true,
+	singleQuote: true,
+	tabWidth: 2,
+	trailingComma: 'es5',
+	useTabs: true,
+	embeddedLanguageFormatting: 'auto',
+	vueIndentScriptAndStyle: false,
+	overrides: [
```

Comment:
are these overrides necessary? They look like the default ones.

Reply:
They were needed in the original POC but looks like this can now be removed. Updated.